src/test/java/com/example/project/page_object/CheckoutPage.java

```diff
@@ -0,0 +1,53 @@
+package com.example.project.page_object;
+
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.PageFactory;
+
+public class CheckoutPage {
+    private WebDriver driver;
+
+    @FindBy(xpath = "//input[@name=\"firstname\"]")
+    private WebElement firstName;
+    @FindBy(xpath = "//input[@name=\"lastname\"]")
+    private WebElement lastName;
+    @FindBy(xpath = "//input[@name=\"streetAddress\"]")
+    private WebElement streetAddress;
+    @FindBy(xpath = "//input[@name=\"city\"]")
+    private WebElement city;
+    @FindBy(xpath = "//body/div[2]/div[4]/div[2]/div[2]/div[1]/div[2]/div[1]/div[1]/div[2]/div[1]/div[1]/div[1]/div[5]/div[2]/div[1]/div[1]/select[1]")
```

Comment:
this is a fragile locator, let's assume that somewhere in the middle new div appeared or existing one disappeared, this will break the locator. Please try to use as few child/parent relations as possible. Ideal locator should point to the exact web element. Like the ones above. If there are no distinct ids or names try to use @class.